README.md

```diff
@@ -94,11 +94,15 @@ MCP Go handles all the complex protocol details and server management, so you ca
   - [Prompts](#prompts)
 - [Examples](#examples)
 - [Extras](#extras)
+  - [Transports](#transports)
   - [Session Management](#session-management)
+    - [Basic Session Handling](#basic-session-handling)
+    - [Per-Session Tools](#per-session-tools)
+    - [Tool Filtering](#tool-filtering)
+    - [Working with Context](#working-with-context)
   - [Request Hooks](#request-hooks)
   - [Tool Handler Middleware](#tool-handler-middleware)
   - [Regenerating Server Code](#regenerating-server-code)
-- [Contributing](/CONTRIBUTING.md)
```

Comment:
Apparently the ToC was not up to date, my formatter automatically formatted it for me on save.